cmd/ingest.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"go/types"
+
+	_ "github.com/lib/pq"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/network"
+	"github.com/stellar/go/support/config"
+	"github.com/stellar/go/support/log"
+	"github.com/stellar/wallet-backend/internal/ingest"
+)
+
+type ingestCmd struct{}
+
+func (c *ingestCmd) Command() *cobra.Command {
+	cfg := ingest.Configs{}
+	cfgOpts := config.ConfigOptions{
+		{
+			Name:        "database-url",
+			Usage:       "Database connection URL.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.DatabaseURL,
+			FlagDefault: "postgres://postgres@localhost:5432/wallet-backend?sslmode=disable",
+			Required:    true,
+		},
+		{
+			Name:        "network-passphrase",
+			Usage:       "Stellar Network Passphrase to connect.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.NetworkPassphrase,
+			FlagDefault: network.TestNetworkPassphrase,
+			Required:    true,
+		},
+		{
+			Name:        "captive-core-bin-path",
+			Usage:       "Path to Captive Core's binary file.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.CaptiveCoreBinPath,
+			FlagDefault: "/usr/local/bin/stellar-core",
+			Required:    true,
+		},
+		{
+			Name:        "captive-core-config-dir",
+			Usage:       "Path to Captive Core's configuration files directory.",
+			OptType:     types.String,
+			ConfigKey:   &cfg.CaptiveCoreConfigDir,
+			FlagDefault: "./internal/ingest/config",
+			Required:    true,
+		},
+		{
+			Name:        "ledger-cursor-name",
+			Usage:       "Name of last synced ledger cursor. Attention: there should never be more than one container running with a same cursor name.",
```

Comment:
Could you elaborate on it? Is this value an enum or should it be a unique value? We should make it more clear and, also remove the `FlagDefault` since it cannot have the same name.

Reply:
It's not an enum, it's just an unique string value used as the key in the `ingest_store` dictionary-like table.

Reply:
I've made the description more clear, made it required and removed the default value. Good call, this will make people more aware that this has to be carefully configured when running parallel ingestion.

Reply:
[72a0dbb](https://github.com/stellar/wallet-backend/pull/14/commits/72a0dbbeeffb8a4c2d127edc74476bd9c69fdf16)